core/src/main/res/values/strings.xml

```diff
@@ -88,6 +88,7 @@
     <string name="author_label">Author(s)</string>
     <string name="language_label">Language</string>
     <string name="url_label">URL</string>
+    <string name="support_label">Support</string>
```

Comment:
In German, the word "support" is translated differently depending on context. I think it would be good to reflect the context in the string name. Something like `support_this_podcast`.

Reply:
Hmm, it's still not 100% specific. Looking at just the name `support_label` and the content, a German translator cannot know that this refers to "Support" (as in "pay") and not "Support" (as in "assistance provided by a company").

Reply:
how about `support_funding_label`?

Reply:
Sounds good 👍